### ISSUE TITLE
Notable People to NPCs

### ISSUE THREAD

**Issue by brunocalado** [drewg13/foundryvtt-scum-and-villainy#47]
The NPC sheet is super useful. I'm even replacing player clocks with it, they can write about the project.

I think it can be even more useful. What do you think about replace the **Notable People** field with a default foundry journal and let us to drag entities to this field? So, you can replace this simple text with links to these notable people in a form of NPCs sheets.

This is useful to put then on a scene with tokens. It's also useful to use Theate and other modules.

![image](https://user-images.githubusercontent.com/4999783/146977705-671ff922-4e61-4cb8-8cc7-d360d4abeec4.png)

Thank you

**Comment by drewg13** [comment]
I can probably leave it as-is for text, but also catch drag and drop events for NPC actors and turn them into links to the sheets. That should be doable. I'll look into it.

**Comment by drewg13** [comment]
Of course, if I just change the field to a TinyMCE field, you can make links to whatever Actors or Items you like without any special code.  So I could just do that.

**Comment by brunocalado** [comment]
Use Cases
- Theate Module
- Token HUD Artwork Button Module
- Add the actor to a map. I'm not even using maps. But, players drawings.
![image](https://user-images.githubusercontent.com/4999783/147294873-46b89e88-601b-4ff8-8909-62653537f484.png)
- Wildcard Module
- You can make long notes to the npc as the game advance. I would even replace Friends for NPCs.
![image](https://user-images.githubusercontent.com/4999783/147294979-e5c3d9af-81f5-425f-99f1-afd156ea189f.png)

**Comment by brunocalado** [comment]
Hi.

I found a npc art pack. I use it, but I was wandering if don't want it too. You can share in the module.
https://ashen-victor.itch.io/sci-fi-character-portraits-poject

I'll edit them to reduce size and convert to webp.

![image](https://user-images.githubusercontent.com/4999783/149574735-db5b2488-5909-4270-974e-65e89ecce61c.png)

**Comment by brunocalado** [comment]
**Rework Friends**
I want to suggest to fully remove the item Friend and Replace this guys for the Actor NPC.

NPCs as actor let lots of modules to work. The one I failed was https://github.com/Aedif/TokenVariants. But, there are more.

**Comment by drewg13** [comment]
I like the portraits.  There wouldn't be a good way to integrate them except as random assignments, the way I did the planets.
The Friends to NPCs idea is workable, but will take more effort, so I'll have to think about that one.

**Comment by drewg13** [comment]
As Foundry doesn't have support for Actors owning other Actors the way Actors can own Items, this is a more complex change than you might think.  But, I already did this with Character Actors "owning" Ship Actors, so the basic idea is already worked out, I'd just have to adapt it.

**Comment by brunocalado** [comment]
> As Foundry doesn't have support for Actors owning other Actors the way Actors can own Items, this is a more complex change than you might think. But, I already did this with Character Actors "owning" Ship Actors, so the basic idea is already worked out, I'd just have to adapt it.

This automation is pretty cool. But, is it really necessary? Maybe just add the links to the compedium npcs to an mce jjournal.

**Comment by drewg13** [comment]
Yeah, changing it to a TinyMCE field is probably the simplest method, but doing that for Friends/Contacts would lose the Friendly/Neutral/Enemy tracking UI.  I'll change the Planet Notable NPCs field for now and look into the more extensive conversion of Friends/Contacts to NPCs later.
